commit 69ac98cb94458779d74b2d2d37535198cb8919d1 (refs/changes/11/811/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2016-10-28T23:39:36+00:00 (2 years, 11 months ago)
    
    MORAY-369 Document that 'retries' cannot be Infinity with node-moray v2
    Reviewed by: Chris Burroughs <chris.burroughs@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index 6dbd88e..5aa5991 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -21,6 +21,10 @@ networking failures.
   updated to use
   [VError.findCauseByName](https://github.com/joyent/node-verror#verrorfindcausebynameerr-name)
   instead of checking the `name` property directly.**
+* `retry.retries` must now be a finite number. Once the number of retries has
+  been exceeded for a backend, the "cueball" module will consider that backend
+  dead, and will take care of periodically checking it to see if it's returned.
+  The recommended and default value is `5`.
 
 **Other changes:**
 
diff --git a/package.json b/package.json
index 89577af..9150387 100644
--- a/package.json
+++ b/package.json
@@ -3,6 +3,7 @@
     "description": "Moray client library",
     "version": "2.0.0",
     "author": "Joyent (joyent.com)",
+    "keywords": [ "moray" ],
     "main": "./lib/index.js",
     "repository": {
         "type": "git",
